hyperion-timber/src/main/java/com/willowtreeapps/hyperion/timber/list/TimberLogListActivity.java

```diff
@@ -113,11 +113,16 @@ private void collectLogs() {
         StringBuilder logs = new StringBuilder();
         for (int i = 0; i < size; i++) {
             LogItem logItem = logItemQueue.getItem(i);
+            String messageWithTag = logItem.tag != null
```

Comment:
Not worth changing but in general, always prefer `==` for conditionals so it's easier to mentally grok. `if null then x else y` vs `if not null then y else x`.